index.md

```diff
@@ -37,8 +37,8 @@ our meetings.
 Where:
 -----
 
-Data Science Initiative Classroom </br>
-Third Floor Shields Library, Rm 360 </br>
+Data Science Initiative Classroom       
+Third Floor Shields Library, Rm 360      
```

Comment:
Won't this put all these on the same line? I think markdown will ignore the line returns.